homeassistant/components/http.py

```diff
@@ -60,11 +66,19 @@ def setup(hass, config):
     development = str(conf.get(CONF_DEVELOPMENT, "")) == "1"
     ssl_certificate = conf.get(CONF_SSL_CERTIFICATE)
     ssl_key = conf.get(CONF_SSL_KEY)
+    allow_cors = conf.get(CONF_ALLOW_CORS, False)
+    cors_headers = conf.get(CONF_CORS_HEADERS, [HTTP_HEADER_ORIGIN,
+                                                HTTP_HEADER_ACCEPT,
+                                                HTTP_HEADER_X_REQUESTED_WITH,
+                                                HTTP_HEADER_CONTENT_TYPE,
+                                                HTTP_HEADER_HA_AUTH])
+    cors_origins = conf.get(CONF_CORS_ORIGINS, ['*'])
```

Comment:
We should never default to match all. That's opening up security without the user being aware.